snippets/go/closed_channel.go

```diff
@@ -0,0 +1,27 @@
+package example
+
+import (
+	"log"
+
+	"github.com/breez/breez-sdk-go/breez_sdk"
+)
+
+func PrepareRedeemOnchainFunds(feeRate uint64) {
+	// ANCHOR: prepare-redeem-onchain-funds
+	satPerVbyte := feeRate
+	destinationAddress := "bc1.."
+	req, err := sdk.PrepareRedeemOnchainFunds(breez_sdk.PrepareRedeemOnchainFundsRequest{SatPerVbyte: satPerVbyte, ToAddress: destinationAddress})
```

Comment:
Break into two lines for readability 
```suggestion
    req := breez_sdk.PrepareRedeemOnchainFundsRequest{SatPerVbyte: satPerVbyte, ToAddress: destinationAddress}
	res, err := sdk.PrepareRedeemOnchainFunds(req)
```

Reply:
Done